operator/controller/integrationroute-controller.yaml

```diff
@@ -22,11 +22,24 @@ spec:
           conditions:
             - type: Ready
               status: "True"
+    - apiVersion: v1
```

Comment:
:question: Why do we need to list ConfigMaps as `childResources`? It doesn't seem like the IntegrationRoute controller creates any CMs.

Reply:
Wow, good catch, this was a leftover from a previous attempt.